cmd/gardener-extension-admission-azure/app/app.go

```diff
@@ -87,6 +90,18 @@ func NewAdmissionCommand(ctx context.Context) *cobra.Command {
 				return err
 			}
 
+			if err := mgr.AddReadyzCheck("informer-sync", gardenerhealthz.NewCacheSyncHealthz(mgr.GetCache())); err != nil {
+				return fmt.Errorf("could not add readycheck for informers: %w", err)
```

Comment:
I can see use of `ready check`(with whitespace) instead of `readycheck`(without whitespace). Please adapt one of them for code consistency.
```suggestion
				return fmt.Errorf("could not add ready check for informers: %w", err)
```

Reply:
I guess readycheck is fine as it is same in other extensions.